tests/php/test-helpers.php

```diff
@@ -1,129 +0,0 @@
-<?php
```

Comment:
This is simply moved to [here](https://github.com/getblocklab/block-lab/pull/402/files#diff-beb131e068d21ed66ea78e959245b576R1).